docs/docs-nav.yml

```diff
@@ -14,6 +14,8 @@
         url: /docs/repeat-failed-test/
       - title: "Report entries"
         url: /docs/report-entries/
+      - title: "Standard input/output"
+        url: /docs/standard-input-output.adoc
```

Comment:
This needs to be written without `.adoc` but with trailing slash. Otherwise the site will break